bookfinder/src/components/Footer.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { FaFacebook, FaInstagram, FaGithub } from "react-icons/fa";
+
+const Footer = () => {
+  return (
+    <div className="top-64 flex h-full w-full flex-col gap-10 bg-gray-300 p-10 xl:flex-row xl:gap-56">
+      <div className="flex flex-col gap-5">
```

Comment:
for small size, all column should be center, you may try to add this here`items-center justify-between md:justify-around`

Reply:
I like the suggestion but most official websites I've seen have their footer content aligned to the left, and I think it looks nicer that way.

Reply:
I didn't think you intentionally aligned left. If you like, you can leave other sizes like this except md size (I think) I don't know what others think?

Reply:
Okay, the md size will be adjusted. Thanks

Reply:
I've implemented the changes you suggested now... Please review it again and let me know what you think.